### PR TITLE
[WFCORE-769] : Boot errors should hide the failure description if the failing operation is not accessible

### PR DESCRIPTION
**Pull request by ehsavoie**
Checking the operation that failed to hide or show failure description.

Jira: https://issues.jboss.org/browse/WFCORE-769

### CODE REVIEW

File: controller/src/main/java/org/jboss/as/controller/BootErrorCollector.java [hunk 1]
```diff
@@ -139,7 +146,8 @@ public OperationStepHandler getReadBootErrorsHandler() {
                 .setRuntimeOnly()
                 .setReplyType(ModelType.LIST)
                 .setReplyParameters(OP_DEFINITION, FAILURE_MESSAGE, FAILED_SVC_AD, MISSING_DEPS_AD, TRANSITIVE_AD).build();
-
+        public static final Set<Action.ActionEffect> DESCRIBE_EFFECTS =
+            Collections.unmodifiableSet(EnumSet.of(Action.ActionEffect.ADDRESS, Action.ActionEffect.READ_CONFIG));
```
Comment (bstansberry) [inline controller/src/main/java/org/jboss/as/controller/BootErrorCollector.java]: This is unused.

**Comment by bstansberry** [comment]
Other than that bit of cruft, looks good.

File: controller/src/main/java/org/jboss/as/controller/BootErrorCollector.java (hunk 1 above)
Comment (ehsavoie) [inline controller/src/main/java/org/jboss/as/controller/BootErrorCollector.java]: Sorry about that :(

**Comment by bstansberry** [comment]
https://github.com/wildfly/wildfly/pull/7624 is a fix for the full integration test; failure is unrelated to this PR.

**Comment by bstansberry** [comment]
retest this please

**Pull request merged**